addons/mail/static/src/components/attachment/attachment_tests.js

```diff
@@ -595,6 +595,34 @@ QUnit.test('close attachment viewer', async function (assert) {
     );
 });
 
+QUnit.test('close attachment viewer before image load', async function (assert) {
+    assert.expect(1);
+
+    await this.start({ hasDialog: true });
+    const attachment = this.env.models['mail.attachment'].create({
+        filename: "test.png",
+        id: 750,
+        mimetype: 'image/png',
+        name: "test.png",
+    });
+    await this.createAttachmentComponent(attachment, {
+        detailsMode: 'hover',
+        isDownloadable: false,
+        isEditable: false,
```

Comment:
These props looks irrelevant for test. Not sure if `detailsMode` is, but maybe it is?